service/test/controllers/usermanagement/UserControllerTest.java

```diff
@@ -325,24 +345,34 @@ public void testupdateLoginTime() {
   }
 
   @Test
-  public void testgetMediaTypes() {
-    PowerMockito.mockStatic(RequestInterceptor.class);
-    when(RequestInterceptor.verifyRequestData(Mockito.anyObject()))
-        .thenReturn("{userId} uuiuhcf784508 8y8c79-fhh");
+  public void testUpdateLoginTimeFailureWithoutUserId() {
+    Map<String, Object> requestMap = new HashMap<>();
+    Map<String, Object> innerMap = new HashMap<>();
+    requestMap.put(JsonKey.REQUEST, innerMap);
+    String data = mapToJson(requestMap);
+    JsonNode json = Json.parse(data);
+    RequestBuilder req =
+        new RequestBuilder().bodyJson(json).uri("/v1/user/update/logintime").method("PATCH");
+    req.headers(headerMap);
+    Result result = route(req);
+    assertEquals(400, result.status());
+  }
+
+  @Test
+  public void testGetMediaTypes() {
```

Comment:
move this from User to some other controller. 

testGetMediaTypes to testGetMediaTypesSuccess